index.js

```diff
@@ -24,9 +24,12 @@ class ServerlessSecretBaker {
       "before:offline:start:end": this.cleanupPackageSecrets.bind(this),
       // For invoke local
       "before:invoke:local:invoke": this.packageSecrets.bind(this),
-      "after:invoke:local:invoke": this.cleanupPackageSecrets.bind(this)
+      "after:invoke:local:invoke": this.cleanupPackageSecrets.bind(this),
```

Comment:
This is a super 'nit', but it might be better to just not include the cleanup step if the option is set, rather than checking the if condition each time it is invoked.  That would keep knowledge about the command line options encapsulated in the constructor.

Reply:
The more I think about this, the more I think that the short lived command should fail if the secret-baker-secrets.json file is already there when it gets invoked.  Then we could have switches to `--ignore` (or skip) that step in case you know what you are doing.  In addition, you could have a `--force` option to tell it to explicitly overwrite and cleanup the file.

Reply:
Ooo yes, I like that.

Reply:
Done!  The assertions is a little wonky.